components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -456,6 +468,7 @@ export function CreateWorkspacePage() {
 
                     <InputField error={errorWsClass}>
                         <SelectWorkspaceClassComponent
+                            selectedConfigurationId={selectedProjectID}
```

Comment:
Thanks for introducing this prop! It's going to be useful in other places as well (like the run prebuild dialog).